modules/github-repository/resources/branch-name-check.yaml

```diff
@@ -0,0 +1,14 @@
+name: 'Branch Naming Check'
+on: pull_request
+
+jobs:
+  branch-naming-rules:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: deepakputhraya/action-branch-name@master
+        with:
+          regex: 'DMVP-[0-9]*' # Regex the branch should match. This example enforces grouping
```

Comment:
can we have the branch name prefix "DMVP"  as variable with this value as default?
so that this can be easily configured if needed in other projects